help-center/faq/cloud-faq.md

```diff
@@ -0,0 +1,83 @@
+---
+id: cloud-faq
+title: Memgraph Cloud - Frequently asked questions
+sidebar_label: Memgraph Cloud
+slug: /faq/cloud
+---
+
+## Pricing and payment
+
+### What is the pricing?
+
+You can estimate the cost of Memgraph Cloud's service by selecting your cloud
+region and instance size with our [Cost
+Calculator](https://cloud.memgraph.com/pricing), or you can check them out at
+[Project rates](/docs/memgraph-cloud/payment#project-rates))
+
+### How can I use a coupon that I got for Memgraph Cloud?
+First, you need to add a credit card to your account. Then you can apply the
+coupon.
+
+### What will happen to my instance after the free trial?
+Instance will be stopped for next 7 days. If you wish to continue, add a payment
+method.
+
+## Projects
+
+### How many projects and snapshots can I create? Why I can’t create more than 3 projects? 
+That is the initial limit for new users. If you want to create more projects,
+[let us know](/help-center).
+
+### Why I can’t create more than 5 snapshots? 
+That is the initial limit for new users. If you want to create more snapshots,
+[let us know](/help-center).
+
+### Is it possible to upgrade a RAM instance?  
+Yes, it is. You can find detailed instructions in [Memgraph Cloud
+documentation](/docs/memgraph-cloud/cloud-projects#resize-a-project).
+
+### I created a 2GB RAM, but Memgraph Labs says there is a total of 1.54GB available. Why is that so?
+A par of RAM is allocated to the operating system and other services. They
+usually take 13-15% of the total RAM. Approximate free RAM is: 
+- 1GB RAM Memgraph Cloud project has about 860 MB free RAM
+- 2GB RAM Memgraph Cloud project has about 1.60 GB free RAM
+- 4GB RAM Memgraph Cloud project has about 3.40 GB free RAM
+- 8GB RAM Memgraph Cloud project has about 6.7 GB free RAM
+- 16GB RAM Memgraph Cloud project has about 14 GB free RAM
+- 32GB RAM Memgraph Cloud project has about 28 GB free RAM
+
+### I've created a new instance, and when I try to connect to it, I get an error: Unable to connect.
```

Comment:
I've created a new project, and when I try to connect to the instance, I get an error: Unable to connect.